src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -72,6 +72,8 @@ public enum Feature {
         STRING_TEMPLATES,
         @JEP(number=445, title="Unnamed Classes and Instance Main Methods")
         UNNAMED_CLASSES,
+        @JEP(number=999, title="Implicit Classes and Instance Main Methods")
+        IMPLICIT_CLASSES,
```

Comment:
I assume you'll change this to 463 before integrating.

Reply:
Missed that. The issue is that Implicit Classes don't have any API and don't show up in the preview page. Fixed.

Reply:
Regarding removal of `UNNAMED_CLASSES` here, there was an API annotated using this constant in `j.l.Class` (`isUnnamedClass()`). I suspect it unfortunately means we need to keep the enum constant as long as this method exists in the bootstrap JDK, i.e. until the bootstrap is switches to JDK 22.

Reply:
Correct